yarn.lock

```diff
@@ -9071,15 +9071,15 @@ gauge@~2.7.3:
     strip-ansi "^3.0.1"
     wide-align "^1.1.0"
 
-geckodriver@^3.0.1:
-  version "3.0.2"
-  resolved "https://registry.yarnpkg.com/geckodriver/-/geckodriver-3.0.2.tgz#6bd69166a24859c5edbc6ece9868339378b6c97b"
-  integrity sha512-GHOQzQnTeZOJdcdEXLuzmcRwkbHuei1VivXkn2BLyleKiT6lTvl0T7vm+d0wvr/EZC7jr0m1u1pBHSfqtuFuNQ==
+geckodriver@^3.0.2:
+  version "3.2.0"
```

Comment:
Is this intentional? `^3.0.2` resolves to `3.2.0`?

Reply:
yes, ^3.0.2 will fetch the latest patch version. though we only require 3.0.2 to solve the cve, there is no breaking changes.